tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/PostResource.java

```diff
@@ -621,4 +623,13 @@ public PostResource(PostService postService, PostValidator postValidator,
 
   }
 
+  @PostMapping(value = "/posts/{postId}/esr-reconciled")
```

Comment:
I think this URI is one of the things that might need a tweak.

Reply:
Thanks @JosephKelly. Do we have any suggested name instead of `esr-reconciled`? `esr-update`?

Reply:
I'm not too sure at the moment.